aem/build.gradle.kts

```diff
@@ -52,14 +67,20 @@ aem {
             dependsOn(":develop")
         }
 
+        environmentDev {
+            reloader {
+                configDir("httpd/conf")
```

Comment:
Why this configuration is still needed?

Reply:
I suggest having this in container:
```kotlin
reload {
                        cleanDir(
                                "/opt/aem/dispatcher/cache/content/example/live",
                                "/opt/aem/dispatcher/cache/content/example/demo"
                        )
                        exec("/usr/local/apache2/bin/httpd -k restart")
                        configDir("httpd/conf")
                    }

```

Reply:
because environment `Reload` is separate thing,  need to know which local files need to be watched

Reply:
added also `containerName` for filtering containers to be reloaded on concrete file changes